resources/roca_lib.brs

```diff
@@ -166,6 +177,14 @@ function __case_execute()
         __state: m.__state
         assert: assert(__util_pass, __util_fail, m.__state)
     }
+
+    ' extra case fields
+    if m.suite.__fields <> invalid then
+        for each item in m.suite.__fields.items()
+            withM.addreplace(item.key, item.value)
```

Comment:
Curious whether we should try to prevent people from replacing `m.describe` or any of the other built-in fields... I don't think so, though. If people want to do that, they can do it at their ⚠️  own risk ⚠️ .

Reply:
It does seem a little strange. My only concern would be that the potential replacement occurs without warning and might lead to non-obvious issues as a result. I don't think it's necessarily blocking for this change, though.

Reply:
I thought it was ok (maybe someone wants to enhance the default methods), though we could raise an error/warning.

Reply:
Yeah agreed, I don't see a reason to prevent people from enhancing it if that's what they want to do.